scripts/decrypt.sh

```diff
@@ -15,8 +15,12 @@ fi
 # Decrypt Google service files (Android and iOS).
 if [[ -n "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" ]]; then
   echo "Decoding google-services files"
-  openssl enc -aes-256-cbc -pbkdf2 -d -k "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" -in ./encryptedFiles/google-services.json.enc -out ./android/app/google-services.json -md md5
-  openssl enc -aes-256-cbc -pbkdf2 -d -k "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" -in ./encryptedFiles/GoogleService-Info.plist.enc -out ./ios/Runner/GoogleService-Info.plist -md md5
+  openssl enc -aes-256-cbc -pbkdf2 -d -k "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" -in ./encryptedFiles/google-services.prod.json.enc -out ./android/app/src/prod/google-services.json -md md5
+  openssl enc -aes-256-cbc -pbkdf2 -d -k "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" -in ./encryptedFiles/google-services.beta.json.enc -out ./android/app/src/beta/google-services.json -md md5
+  openssl enc -aes-256-cbc -pbkdf2 -d -k "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" -in ./encryptedFiles/google-services.dev.json.enc -out ./android/app/src/dev/google-services.json -md md5
+  openssl enc -aes-256-cbc -pbkdf2 -d -k "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" -in ./encryptedFiles/GoogleService-Info.prod.plist.enc -out ./ios/config/prod/GoogleService-Info.plist -md md5
+  openssl enc -aes-256-cbc -pbkdf2 -d -k "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" -in ./encryptedFiles/GoogleService-Info.beta.plist.enc -out ./ios/config/beta/GoogleService-Info.plist -md md5
+  openssl enc -aes-256-cbc -pbkdf2 -d -k "$ENCRYPTED_GOOGLE_SERVICE_PASSWORD" -in ./encryptedFiles/GoogleService-Info.dev.plist.enc -out ./ios/config/dev/GoogleService-Info.plist -md md5
```

Comment:
Ahh great point, I'll check that out tomorrow! 😄 I'll add two other variables for dev and beta

Reply:
Also, for dev and beta, it's the same Google service files. I figured that maybe it won't really matter if we have a firebase project for 5 or 6 person (dev) so I choose to drop all dev analytics entry directly in beta project. This is by design but if you prefer that we have three separate firebase project it will take me 5min to do :smile: What do you think?